### ISSUE TITLE
Edge isn't supported?

### ISSUE THREAD

**Issue by georgeedwards** [angular/material2#7588]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
Edge to render angular apps with material apps in the same way as Chrome and other browsers

#### What is the current behavior?

I know this sounds slightly odd, as I am sure there are tests somewhere? However, when adding angular material 2 to a cli project, the whole app stops rendering on Edge, where it is fine in Chrome, I also checked against [https://material.angular.io/](https://material.angular.io/) and the regular angular.io site, but this is what I get:

[![enter image description here][1]][1]

I have refreshed numerous times, restarted my system. Edge works as expected on all non-angular/material sites I tried. 

Here is the log:

[![enter image description here][2]][2]

  [2]: https://i.stack.imgur.com/rMVLW.png

#### What are the steps to reproduce?
Open the angular.material.io site in an affected browser. 

#### What is the use-case or motivation for changing an existing behavior?
Meeting the stated browser support levels

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
```
@angular/material@2.0.0-beta.12
Angular 4.2.4
Microsoft Edge 40.15063.0.0
Microsoft EdgeHTML 15.15063
```

#### Is there anything else we should know?

  [1]: https://i.stack.imgur.com/BnLVY.png

**Comment by alinaMihai** [comment]
I found this surprising so I also checked it out.
I had no problem viewing https://material.angular.io on this version of Edge:
Microsoft Edge 25.10586.672.0
Microsoft EdgeHTML 13.10586

**Comment by crisbeto** [comment]
Edge is supported, however we have a fallback animation. That being said the fallback animation doesn't work at the moment and will be fixed by https://github.com/angular/material2/pull/7599.

**Comment by danwulff** [comment]
Also seems to load just fine (barring the other linked issues with the spinner and whatnot) with:
Microsoft Edge 40.15063.0.0
Microsoft EdgeHTML 15.15063
Windows 10 Pro v. 1703, build 15063.632

**Comment by josephperrott** [comment]
Closing as fixed by #7599

**Comment by maiconparra** [comment]
Estou com um problema parecido só que com as tabelas expasivas do material. Tando no Edge quanto no Safari. Uma das tabelas funciona, no entanto as outras não. Não há problemas na codificação pois se não também não funcionaria no Chrome, Mozila e Opera. Fique sabendo atravez de uma pesquisa que o Edge não terá 50 elementos do Chromium e por este algumas funcionalidades não funcionaram como esperado. Se alguém tiver passado par este problema eu ficaria muito grato de saber como alcansou a solução.
